docs/discovery.md

```diff
@@ -32,28 +32,72 @@ This is trivial, simply do:
 HTTP GET https://platform.example/organization/123/did.json => didDocument.json
 ```
 
-2. Review the `alsoKnownAs` section of the did document.
+2. Review the `alsoKnownAs`, `assertionMethod` and `authentication` sections of the did document.
 
-```json
-{
-  "alsoKnownAs": [
-    "did:example:123",
-    "did:key:z6Mk...",
-    "did:web:another.platform.example:organization:456"
-  ]
-}
-```
+#### `alsoKnownAs`
+
+> This relationship is a statement that the subject of this identifier is also identified by one or more other identifiers.
+
+See [alsoKnownAs](https://www.w3.org/TR/did-core/#dfn-alsoknownas).
+
+This entry MUST be present.
+This entry MUST have a first element that is the base URL for a VC-API that supports presentation exchange.
+
+#### `assertionMethod`
+
+The set of supported `assertionMethod` DID URLs for the organization.
+See [assertionMethod](https://www.w3.org/TR/did-core/#assertion).
 
-Resolve each of these DIDs.
 
-The set of supported `assertionMethod` DID URLs for the organization is the aggregation of the `assertionMethod` for each listed DID.
+#### `authentication`
 
-The set of supported `authentication` DID URLs for the organization is the aggregation of the `authentication` for each listed DID.
+The set of supported `authentication` DID URLs for the organization.
+See [authentication](https://www.w3.org/TR/did-core/#authentication).
+
+The `didDocument` MAY contain a `verificatonMethod` section, which MAY be used to support `did:web` based verification relationships.
+
+The `didDocument` MUST NOT contain `verificatonMethod`'s where the controller is different from the DID Subject.
```

Comment:
```suggestion
The `didDocument` MUST NOT contain `verificationMethods` where the controller is different from the DID Subject.
```
Style choice -- either "`verificationMethod`s" or "`verificationMethods`" (that is, ``` `verificationMethod`s``` or ``` `verificationMethods` ```); I think the latter is better.